builtin/staker/aggregation/aggregation.go

```diff
@@ -62,31 +54,23 @@ func (a *Aggregation) NextPeriodTVL() (uint64, error) {
 // 3. Move ExitingVET to WithdrawableVET
 // return a delta object
 func (a *Aggregation) renew() (*globalstats.Renewal, error) {
-	if a.ExitingVET > a.LockedVET+a.PendingVET {
```

Comment:
removed checks here since `a.Locked.Add(a.Pending)` then `a.Locked.Sub(a.Exiting)` and `Sub` checks underflow